gulp/plugins/gulp-react-docgen.js

```diff
@@ -47,10 +47,9 @@ module.exports = (filename) => {
 
       cb()
     } catch (err) {
-      this.emit(
-        'error',
-        new gutil.PluginError(pluginName, err)
-      )
+      const pluginError = new gutil.PluginError(pluginName, err)
+      pluginError.message += `\nFile: ${file.path}.`
+      this.emit('error', pluginError)
```

Comment:
If docgen fails to generate docs for a file, append the filename to the error message so we know which one failed.